### DISCUSSION TITLE
Application approved but no benefits available

### DISCUSSION THREAD

**Discussion by MarvinRucinski** [community/community#147080] [GitHub Education]
### Hi everyone,

My application got approved a few days ago but GitHub Pro is not active. The FAQ said to raise a discussion here.

**Comment by MarvinRucinski** [comment]
I didn't get an email but I see that it is approved on the webpage.

**Comment by Er-Ayush1** [comment]
How to contact GitHub support

**Comment by davevad93** [comment]
Hi @MervinDelfin ,

I assume your status is "`Congrats! Your request for @MervinDelfin was approved and your benefits are now available.`" and I also see you have the `PRO` badge, which means should have access to your benefits, if you're experiencing problems in redeeming them, try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.
